### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by senlin**
I have taken a first stab at updating the readme.html file from an issue that has been open since a few months already (#33)

It still needs a paragraph of text from our Founder @scottybo 

## Screenshots (if appropriate):
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by nylen** [comment]
Looks good to me!

Maybe we should use the logomark with text here?



@johnalarcon has done the same thing in his PR for the installation screens, #265.

**Comment by senlin** [comment]
> Maybe we should use the logomark with text here?

Excellent idea! I will change it right away!

**Comment by nylen** [comment]
> Hmmm, I can link to the file, but it is not (yet) located where it says it is located in that PR. Maybe because that PR is still open?

Yes, that's right.  For now, you'll need to add the file to this PR too.

**Comment by nylen** [comment]
Oh, also, the file in that PR is 239 x 46 pixels, which is not nearly large enough.  As we've done elsewhere, you can just use the SVG from https://docs.classicpress.net/press-and-branding/ directly.

**Comment by senlin** [comment]
Added the svg logo, but no idea why it does not show on my previews?

File: src/readme.html [hunk 1]
```diff
@@ -3,96 +3,96 @@
...
-<p>If you are updating from version 2.7 or higher, you can use the automatic updater:</p>
+<p>If you are updating from beta version 1.0.0 or higher, you can use the automatic updater:</p>
 <ol>
 	<li>Open <span class="file"><a href="wp-admin/update-core.php">wp-admin/update-core.php</a></span> in your browser and follow the instructions.</li>
-	<li>You wanted more, perhaps? That&#8217;s it!</li>
+	<li>That&#8217;s it!</li>
 </ol>
 
-<h3>Updating Manually</h3>
+<h3>Updating Manually via (S)FTP or the admin panel of the web host (such as cPanel)</h3>
```
Comment (nylen) [inline src/readme.html]: This is very long for a heading.  I'd move the added text out into a `p` tag.

File: src/readme.html [hunk 2]
```diff
@@ -3,96 +3,96 @@
...
 </h1>
-<p style="text-align: center">Semantic Personal Publishing Platform</p>
+<p style="text-align: center">The business-focused CMS.<br>Powerful. Versatile. Predictable.</p>
 
-<h2>First Things First</h2>
-<p>Welcome. WordPress is a very special project to me. Every developer and contributor adds something unique to the mix, and together we create something beautiful that I&#8217;m proud to be a part of. Thousands of hours have gone into WordPress, and we&#8217;re dedicated to making it better every day. Thank you for making it part of your world.</p>
-<p style="text-align: right">&#8212; Matt Mullenweg</p>
+<h2>A word from our Founder</h2>
+<p>[we need a paragraph intro text from Scott for this]</p>
+<p style="text-align: right">&#8212; Scott Bowler</p>
```
Comment (nylen) [inline src/readme.html]: It's probably going to be best to wait for a separate PR for a word from the founder.

File: src/readme.html [hunk 3]
```diff
@@ -3,96 +3,96 @@
...
+	<dt><a href="https://www.classicpress.net/democracy/">ClassicPress Democracy</a></dt>
+		<dd>Last but not least it is good to read about the ClassicPress Democracy. ClassicPress is a democratic community-led fork of WordPress that enables all stakeholders to shape the direction that the project takes. This page aims to explain how we handle this process to ensure that power doesn’t become centralised and that every voice can be heard.</dd>
 </dl>
 
 <h2>Final Notes</h2>
-<ul>
-	<li>If you have any suggestions, ideas, or comments, or if you (gasp!) found a bug, join us in the <a href="https://forums.classicpress.net/c/support">Support Forums</a>.</li>
-	<li>WordPress has a robust plugin <abbr title="application programming interface">API</abbr> that makes extending the code easy. If you are a developer interested in utilizing this, see the <a href="https://developer.wordpress.org/plugins/">Plugin Developer Handbook</a>. You shouldn&#8217;t modify any of the core code.</li>
+<p>If you have any suggestions, ideas, or comments, or if you found a bug, join us in the <a href="https://forums.classicpress.net/c/support">Support Forums</a>.</p>
 </ul>
```
Comment (nylen) [inline src/readme.html]: The opening tag for this `ul` was removed, so the closing `</ul>` needs to be removed too.

File: src/readme.html (hunk 2 above)
Comment (senlin) [reply src/readme.html]: Yes, exactly, so that is why I already filled in the blanks, bit silly to keep Matt's name there, no?

**Comment by senlin** [comment]
> Let's pick one or the other (CSS background applied to the `a` tag, or an `img` tag by itself). I suspect that an `img` tag by itself is going to be easier to maintain and work with.

I agree, but if I remove the styling from the `a` tag, doesn't that mess up @johnalarcon's PR then?

File: src/readme.html (hunk 3 above)
Comment (senlin) [reply src/readme.html]: sorry about that

File: src/readme.html (hunk 1 above)
Comment (senlin) [reply src/readme.html]: You're right. moved to its own paragraph.

**Comment by senlin** [comment]
Travis is showing the following [error](https://travis-ci.org/ClassicPress/ClassicPress/jobs/469393469#L923-L931):

```
One or more files were modified when running the precommit checks:
src/wp-admin/images/cp-logo-gradient.png
tests/qunit/fixtures/wp-api-generated.js
Please run `grunt precommit` and commit the results.
Fatal error: Modified files detected during precommit checks!
The command "grunt $WP_TRAVISCI" exited with 3.
```

> Please run `grunt precommit` and commit the results.

No idea how to do that.

File: src/readme.html [hunk 4]
```diff
@@ -3,96 +3,96 @@
...
-<p>WordPress can <a href="https://codex.wordpress.org/Importing_Content">import from a number of systems</a>. First you need to get WordPress installed and working as described above, before using <a href="wp-admin/import.php" title="Import to WordPress">our import tools</a>.</p>
+<h2>Migrating from WordPress</h2>
+<p>With the ClassicPress migration plugin, it is very easy to migrate from WordPress to ClassicPress. Please follow the <a href="https://docs.classicpress.net/installing-classicpress/#migrate-classicpress">documentation</a> for step-by-step instructions.</p>
 
 <h2>System Requirements</h2>
 <ul>
-	<li><a href="https://secure.php.net/">PHP</a> version <strong>5.2.4</strong> or higher.</li>
-	<li><a href="https://www.mysql.com/">MySQL</a> version <strong>5.0</strong> or higher.</li>
+	<li><a href="https://secure.php.net/">PHP</a> version <strong>5.6</strong> or higher.</li>
+	<li><a href="https://www.mysql.com/">MySQL</a> version <strong>5.6</strong> or higher.</li>
```
Comment (nylen) [inline src/readme.html]: We're still on MySQL v5.0.

File: src/readme.html [hunk 5]
```diff
@@ -3,96 +3,96 @@
...
+	<li><a href="https://letsencrypt.org/">HTTPS</a> support.</li>
```
Comment (nylen) [inline src/readme.html]: A HTTPS-enabled server is not required to use ClassicPress.

You're probably thinking of the fact that all _outgoing_ API connections must be made through HTTPS.  There's no need to mention this in the readme.  I have never seen a server that does not support this, but WordPress can run in such places so we added a check to the migration plugin.

File: src/readme.html [hunk 6]
```diff
@@ -3,96 +3,96 @@
...
 </ul>
 
 <h3>Recommendations</h3>
 <ul>
 	<li><a href="https://secure.php.net/">PHP</a> version <strong>7.2</strong> or higher.</li>
-	<li><a href="https://www.mysql.com/">MySQL</a> version <strong>5.6</strong> or higher.</li>
+	<li><a href="https://www.mysql.com/">MySQL</a> version <strong>5.7</strong> or higher.</li>
 	<li>The <a href="https://httpd.apache.org/docs/2.2/mod/mod_rewrite.html">mod_rewrite</a> Apache module.</li>
-	<li><a href="https://wordpress.org/news/2016/12/moving-toward-ssl/">HTTPS</a> support.</li>
```
Comment (nylen) [inline src/readme.html]: Let's leave this recommendation in place.  See above.

File: src/readme.html [hunk 7]
```diff
@@ -3,96 +3,96 @@
...
+<p>ClassicPress has no multi-million dollar marketing campaign or celebrity sponsors, but we do have something even better&#8212;you. If you enjoy ClassicPress please consider telling a friend, setting it up for someone less knowledgable than yourself, or writing the author of a media article that overlooks us.</p>
 
-<p>WordPress is the official continuation of <a href="http://cafelog.com/">b2/caf&#233;log</a>, which came from Michel V. The work has been continued by the <a href="https://wordpress.org/about/">WordPress developers</a>. If you would like to support WordPress, please consider <a href="https://wordpress.org/donate/" title="Donate to WordPress">donating</a>.</p>
+<p>ClassicPress is the official business focused CMS and is a hard fork of WordPress 4.9.8. WordPress istself is a fork of <a href="https://www.whoishostingthis.com/resources/b2-cafelog/">b2/caf&#233;log</a> from Michel Valdrighi. If you would like to support ClassicPress, please consider <a href="https://donate.classicpress.net/" title="Donate to ClassicPress">donating</a>.</p>
```
Comment (nylen) [inline src/readme.html]: Typo: `istself` → `is itself`

**Comment by nylen** [comment]
Part of the problems with Travis here will be resolved in #309.

I can fix the other file it's complaining about, but it will change the history of this PR from what you have locally so it is probably best to wait until we're happy with the rest of the PR first.

File: src/readme.html (hunk 5 above)
Comment (senlin) [reply src/readme.html]: Yes, that was exactly my line of thinking, sorry for mixing it up and thanks for explaining it.

File: src/readme.html (hunk 6 above)
Comment (senlin) [reply src/readme.html]: okay

File: src/readme.html (hunk 4 above)
Comment (senlin) [reply src/readme.html]: changed it back

File: src/readme.html (hunk 2 above)
Comment (senlin) [reply src/readme.html]: so leaving this for Scott then, he can send something that we can then add as a separate PR.

File: src/readme.html (hunk 7 above)
Comment (senlin) [reply src/readme.html]: done too

**Comment by senlin** [comment]
Regarding Travis, thanks for your work, but the 7.1 test still seems to be failing.

File: src/readme.html (hunk 2 above)
Comment (nylen) [reply src/readme.html]: @senlin whatever we get done in this PR is probably going to ship in v1, so it should not be "we need a paragraph from Scott for this".  Can you work with @scottybo to get something we can drop in here, or should we remove this section?

File: src/readme.html [hunk 8]
```diff
@@ -3,57 +3,59 @@
...
+	<a href="https://www.classicpress.net/">ClassicPress</a>
 </h1>
-<p style="text-align: center">Semantic Personal Publishing Platform</p>
+<p style="text-align: center">The business-focused CMS.<br>Powerful. Versatile. Predictable.</p>
 
-<h2>First Things First</h2>
-<p>Welcome. WordPress is a very special project to me. Every developer and contributor adds something unique to the mix, and together we create something beautiful that I&#8217;m proud to be a part of. Thousands of hours have gone into WordPress, and we&#8217;re dedicated to making it better every day. Thank you for making it part of your world.</p>
-<p style="text-align: right">&#8212; Matt Mullenweg</p>
+<h2>A word from our Founder</h2>
+<p>[we need a paragraph intro text from Scott for this]</p>
```
Comment (nylen) [inline src/readme.html]: @senlin whatever we get done in this PR is probably going to ship in v1, so it should not be "we need a paragraph from Scott for this".  Can you work with @scottybo to get something we can drop in here, or should we remove this section?

Comment (senlin) [reply src/readme.html]: @nylen when I commited this more than a month ago it was still a valid "we need a paragraph from Scott for this".
I will see what I can get from him and update this ticket accordingly. Do we know when v1 ships?

Comment (nylen) [reply src/readme.html]: v1 can ship when all the things in https://github.com/ClassicPress/ClassicPress/milestone/5 and https://github.com/ClassicPress/ClassicPress/milestone/3 are done.  This PR is on the list, thanks for your work on it so far.

We don't have an ETA right now  Progress has slowed down due to the holidays and me having to focus more on client work.

File: src/readme.html (hunk 5 above)
Comment (nylen) [reply src/readme.html]: Let's move this line below, to Recommendations.  It can say "Serve your site via HTTPS with a [free SSL certificate](https://letsencrypt.org/)".

**Comment by nylen** [comment]
Intro paragraph is in progress here:  https://forums.classicpress.net/t/feedback-requested-intro-paragraph-for-readme-file/711
